package-lock.json

```diff
@@ -4909,7 +4924,6 @@
             "json5": "^2.1.0",
             "lodash": "^4.17.13",
             "resolve": "^1.3.2",
-            "semver": "^5.4.1",
```

Comment:
Can we remove some of these unnecessary `package-lock.json` changes? We should only need the ones for the new types.

Reply:
The package-lock.json file is automatic. If we modify it from the generated version it's just going to reset it isn't it?

Reply:
Done.

> The package-lock.json file is automatic. If we modify it from the generated version it's just going to reset it isn't it?

I usually use yarn so I may not be correct, but using different versions of `npm` when modifying the lock might generate different changes. I know with `yarn` sometimes I see integrity hashes being added for packages that remained unchanged.